src/main.rs

```diff
@@ -141,6 +158,12 @@ fn main() {
             if let Err(handled_pr) = handle_pull_request(pr, &bitbucket, &config.teamcity, &fanout) {
                 println!("{}{}", prefix(2), handled_pr);
             }
+
+            match &schedule {
+                &Some(ref sch) => sleep_duration = (sch.next_utc().unwrap() - UTC::now()).to_std().unwrap(),
+                _ => continue
+            }
```

Comment:
There is a bug in your pattern matching. If `schedule` is `None`, then the loop will continue immediately without sleeping.

You can do nothing on a pattern match arm with something like `None => {}`.

```rust
let sleep_duration = match schedule {
    Some(ref sechedule) => (sechedule.next_utc().unwrap() - UTC::now()).to_std().unwrap(),
    None => fixed_interval.unwrap()
};
```